src/pages/office/office.ts

```diff
@@ -570,7 +570,9 @@ export class OfficePage extends PracticeableBasePageComponent {
   }
 
   completeTest() {
-    this.store$.dispatch(new CompleteTest());
+    if (!this.isPracticeMode) {
+      this.store$.dispatch(new CompleteTest());
```

Comment:
Does this cover tests completed by terminations?

Reply:
Of course it does, ignore me,

Reply:
Yes, just checked and terminations result in no upload and reset of the fake journal page.